packages/drivers/local-driver/src/localDocumentDeltaConnection.ts

```diff
@@ -59,8 +61,8 @@ export class LocalDocumentDeltaConnection extends DocumentDeltaConnection {
 		return deltaConnection;
 	}
 
-	constructor(socket: Socket, documentId: string) {
-		super(socket, documentId, new TelemetryNullLogger());
+	constructor(socket: Socket, documentId: string, logger?: ITelemetryBaseLogger) {
+		super(socket, documentId, createChildLogger({ logger }));
```

Comment:
Nit: I think this could actually be changing behavior slightly, not 100% sure. Inside `createChildLogger()`, even if the `logger` here is undefined, this will not qualify as undefined props but an empty object, right? If so, it will go through a slightly different path and end up returning a `ChildLogger` with a `TelemetryBaseNullLogger` as base instead of directly a `TelemetryNullLogger`.

Reply:
that is correct, but i think the null logger here was wrong, as it will lead to us loosing logs in tests. i think the change is minor. happy to put in a different review if you think it is necessary.

Reply:
> happy to put in a different review if you think it is necessary.

Not really, if there was intention behind the change I don't have objections.